src/txn/txn.c

```diff
@@ -1139,12 +1148,15 @@ __txn_resolve_prepared_op(WT_SESSION_IMPL *session, WT_TXN_OP *op, bool commit,
             WT_ERR(__wt_upd_alloc_tombstone(session, &tombstone, &not_used));
 #ifdef HAVE_DIAGNOSTIC
             WT_WITH_BTREE(session, op->btree,
-              ret = __wt_row_modify(
-                cbt, &cbt->iface.key, NULL, tombstone, WT_UPDATE_INVALID, false, false));
+              ret = btree->type == BTREE_ROW ?
+                __wt_row_modify(
+                  cbt, &cbt->iface.key, NULL, tombstone, WT_UPDATE_INVALID, false, false) :
+                __wt_col_modify(cbt, cbt->recno, NULL, tombstone, WT_UPDATE_INVALID, false, false));
 #else
             WT_WITH_BTREE(session, op->btree,
-              ret =
-                __wt_row_modify(cbt, &cbt->iface.key, NULL, tombstone, WT_UPDATE_INVALID, false));
+              ret btree->type == BTRE_ROW ?
```

Comment:
Missing of `=`

Reply:
That is bizarre; I built it right before I pushed it...

Reply:
oh I see, it's in a non-diagnostic ifdef